### ISSUE TITLE
Footer needs unique ARIA label

### ISSUE THREAD

**Issue by cjg89** [UCF/UCF-WordPress-Theme#87]
**Describe the bug**

When combined with the UCF Footer, or when any other `<footer>` element is in use on any page using this theme, non-distinguishable landmark warnings get triggered by accessibility scanners.

**Steps to Reproduce**

Inspect a page in SiteImprove with the UCF Footer and a site-specific footer.

**Expected Behavior**

The `<footer>` defined in the footer.php template part should have a unique ARIA label.
https://github.com/UCF/UCF-WordPress-Theme/blob/master/template-parts/footer.php#L9

**Possible Fix**

Add an `aria-label` attribute.

**Your environment:**

n/a

**Additional context**

See also: https://github.com/UCF/UCF-Footer-Plugin/issues/18
